libs/shared/base-abstract-classes/src/service/handlers/handler.service.ts

```diff
@@ -43,12 +44,18 @@ export abstract class HandlerService<T extends TableRow> extends SubscriptionHan
     this.resetStatus();
     this.isNew = isNew;
 
+    const currentSelected = this._selected;
+
     if (typeof id === 'object') {
       this._selected = JSON.stringify(id);
     } else {
       this._selected = `${id}`;
     }
 
+    if (currentSelected === this._selected) {
+      this.forceReload = true;
+    }
+
```

Comment:
can you please leave a comment explaining why this is needed and when it will be triggered?

Reply:
done